src/sagas/recommended.js

```diff
@@ -152,11 +152,12 @@ export function* fetchRecommended(action) {
     // Append predefined, constant recommentations.
     if (variant === AB_RECOMMEND_NEW) {
       data.feeds.splice(data.feeds.length - 1, 0, [
-        { content_type: CT_SOURCE, uid: 'itcVAcFn' },         // Maamar Ha-Arvut
+        /*{ content_type: CT_SOURCE, uid: 'itcVAcFn' },/*     // Maamar Ha-Arvut
```

Comment:
are you make it special (use /* instead */)?

Reply:
Fixed.